packages/pangea-node-sdk/tests/unit/config.test.ts

```diff
@@ -10,7 +10,7 @@ const serviceName = "serviceName";
 
 it("insecure true, environment local", async () => {
   const config = new PangeaConfig({ domain: domain, insecure: true, environment: ConfigEnv.LOCAL });
-  const request = new PangeaRequest(serviceName, token, config, false);
+  const request = new PangeaRequest(serviceName, token, config, undefined);
```

Comment:
The configID is optional, so it shouldn't need to be included, the value will already be `undefined`.

Reply:
fixed too